### PR TITLE
Add crashlog instructions for non-vanilla launchers

### PR DESCRIPTION
**Pull request by KTrain5169**
Would it just be better to put a GitHub Gist with instructions or smth?
(Obviously vanilla launcher steps would still be using KTibow's video)

### CODE REVIEW

File: src/interaction-handlers/ticket/categorizeCrash.ts [hunk 1]
```diff
@@ -37,11 +37,14 @@ export class ButtonHandler extends InteractionHandler {
         );
       case "launcher":
         return channel.send(`You'll need to send your crash log. Please follow these instructions:
-1. Open your SkyClient folder. See this: https://youtu.be/BHIM2htfMk8
+1. Open your SkyClient folder. If you use the vanilla launcher, see this: https://youtu.be/BHIM2htfMk8
```
Comment (RuinTD) [inline src/interaction-handlers/ticket/categorizeCrash.ts]: New users probably won't know what "vanilla launcher" is. "Minecraft Launcher" is probably preferred here.
```suggestion
1. Open your SkyClient folder. If you use Minecraft Launcher, see this: https://youtu.be/BHIM2htfMk8
```

File: src/interaction-handlers/ticket/categorizeCrash.ts [hunk 2]
```diff
@@ -37,11 +37,14 @@ export class ButtonHandler extends InteractionHandler {
...
 2. - **If you saw a "View Crash Report" button in the launcher:**
      Go to the \`crash-reports\` folder and upload the most recent file here.
    - **If you didn't see the button:**
      Go to the \`logs\` folder and upload the file called \`latest\` or \`latest.log\`.
+   - **If you use Prism, Modrinth (app) or another non-vanilla launcher:***
```
Comment (RuinTD) [inline src/interaction-handlers/ticket/categorizeCrash.ts]: We only "officially" support Prism and Modrinth. This is probably better phrasing:
```suggestion
   - **If you use the Prism or Modrinth Launcher:***
```

File: src/interaction-handlers/ticket/categorizeCrash.ts [hunk 3]
```diff
@@ -37,11 +37,14 @@ export class ButtonHandler extends InteractionHandler {
...
+     Click the \`Folder\` button in your launcher and open the \`logs\` folder.
+     Then, upload the \`fml-client-latest\` file into the channel.
```
Comment (RuinTD) [inline src/interaction-handlers/ticket/categorizeCrash.ts]:
```suggestion
     Then upload the \`fml-client-latest\` file here.
```

**Review by RuinTD — APPROVED** [review]

**Review by RuinTD — CHANGES_REQUESTED** [review]
Too wordy. What was there before was sufficient.

**Review by RuinTD — APPROVED** [review]

**Comment by RuinTD** [comment]
I'm going to refine this in main branch.

**Comment by KTrain5169** [comment]
No longer necessary with the new code in place. Will probably open another PR to refine instructions there.
